app/views/applications/show.html.erb

```diff
@@ -1,4 +1,7 @@
 <h1><%= @application.name %></h1>
-<h3><%= @application.address %> years old</h3>
+<h3><%= @application.street %></h3>
+<h3><%= @application.city %></h3>
```

Comment:
In the 1st user story, it states "Full Address of the Applicant including street address, city, state, and zip code". Since I worked on the show page, I can combine all the address components